Measurements/Measurements.ipynb

```diff
@@ -572,6 +572,43 @@
+    }
+   },
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Solution:**\n",
+    "Let ${|E_a⟩, |E_b⟩}$ be a measurement with two outcomes $a$ and $b$, which we identify with the answers, i.e., \"a\" = \"state was |0⟩\" and \"b = state was |+⟩\". Then we define\n",
+    "<li>\n",
+    "<ul>$P(a|0)$ = probability to observe first outcome given that the state was $|0⟩$</ul>\n",
+    "<ul>$P(b|0)$ = probability to observe second outcome given that the state was $|0⟩$</ul>\n",
+    "<ul>$P(a|+)$ = probability to observe first outcome given that the state was $|+⟩$</ul>\n",
+    "<ul>$P(b|+)$ = probability to observe second outcome given that the state was $|+⟩$</ul>\n",
+    "</li>\n",
+    "The task is to maximize the probability to be correct on a single shot experiment, which is the same as to minimize the probability to be wrong (on a single shot). Assuming uniform prior, i.e., $P(+) = P(0) = \\frac{1}{2}$, we get<br>\n",
+    "$P_{correct} = P(0) P(a|0) + P(+) P(b|+) = \\frac{1}{2} (P(a|0) + P(b|+))$\n",
+    "\n",
+    "Assuming a von Neumann measurement of the form\n",
+    "<li>\n",
+    "<ul>$E_a = R_y(2\\alpha) \\begin{bmatrix} 1\\\\0\\end{bmatrix} = \\begin{bmatrix}cos(\\alpha) \\\\sin(\\alpha)\\end{bmatrix}$ and</ul>\n",
+    "<ul>$E_b = R_y(2\\alpha) \\begin{bmatrix} 0\\\\1\\end{bmatrix} = \\begin{bmatrix}-sin(\\alpha) \\\\ cos(\\alpha)\\end{bmatrix}$, we get</ul>\n",
+    "<ul>$P(a|0) = |⟨E_a|0⟩|² = cos²(\\alpha)$,</ul>\n",
```

Comment:
You can use `\cos` in the formulas to typeset cosine in non-italic font, and `^2` instead of Unicode character ² - I had to use what I could in the VS projects, but there is no reason not to take advantage of the LaTeX formatting when we have it available in the Notebooks :-)